plugins/by-name/overseer-nvim/default.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  ...
+}:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+  inherit (lib) types;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "overseer-nvim";
```

Comment:
Genuine question, why not `overseer` in this case ?

Reply:
Didn't know what the preference was for following repo name or lua setup name

Reply:
The convention we have been using until now mostly correspond to the lua name, i.e. plugin name without the `.nvim` suffix.

Reply:
Sounds good I'll change these init prs to do that